src/gpu/sentry_gpu_common.c

```diff
@@ -0,0 +1,93 @@
+#include "sentry_gpu.h"
+#include "sentry_string.h"
+
+char *
+sentry__gpu_vendor_id_to_name(unsigned int vendor_id)
+{
+    switch (vendor_id) {
+    case 0x10DE:
+        return sentry__string_clone("NVIDIA Corporation");
+    case 0x1002:
+        return sentry__string_clone("Advanced Micro Devices, Inc. [AMD/ATI]");
+    case 0x8086:
+        return sentry__string_clone("Intel Corporation");
+    case 0x106B:
+        return sentry__string_clone("Apple Inc.");
+    case 0x1414:
+        return sentry__string_clone("Microsoft Corporation");
+    case 0x5143:
+        return sentry__string_clone("Qualcomm");
+    case 0x1AE0:
+        return sentry__string_clone("Google");
+    case 0x1010:
+        return sentry__string_clone("VideoLogic");
+    case 0x1023:
+        return sentry__string_clone("Trident Microsystems");
+    case 0x102B:
+        return sentry__string_clone("Matrox Graphics");
+    case 0x121A:
+        return sentry__string_clone("3dfx Interactive");
+    case 0x18CA:
+        return sentry__string_clone("XGI Technology");
+    case 0x1039:
+        return sentry__string_clone("Silicon Integrated Systems [SiS]");
+    case 0x126F:
+        return sentry__string_clone("Silicon Motion");
+    default:
+        return sentry__string_clone("Unknown");
```

Comment:
How about something like this so we can look at those by usage and expand?
```suggestion
        return sentry__string_clone("Unknown - " + vendor_id);
```